python/ql/test/library-tests/frameworks/flask/response_test.py

```diff
@@ -57,6 +57,11 @@ def html7():  # $requestHandler
     resp.set_data("<h1>hello</h1>")  # $ MISSING: responseBody="<h1>hello</h1>"
     return resp  # $ SPURIOUS: HttpResponse mimetype=text/html responseBody=resp
 
+@app.route("/html8")  # $routeSetup="/html8"
+def html8():  # $requestHandler
+    resp = Response(response="<h1>hello</h1>")  # $HttpResponse mimetype=text/html responseBody="<h1>hello</h1>"
+    return resp  # $ SPURIOUS: HttpResponse mimetype=text/html responseBody=resp
+
```

Comment:
I would expect this test case to appear right after `html4`, which, ironically, would entail a bunch of renames...

Reply:
(I seem unable to construct a suggestion for this..)

Reply:
Haha, ok, will do.